cms/djangoapps/contentstore/views/tests/test_transcripts.py

```diff
@@ -628,6 +629,58 @@ def test_check_youtube(self):
             }
         )
 
+    @patch('xmodule.video_module.transcripts_utils.requests.get', side_effect=mocked_requests_get)
+    def test_check_youtube_with_transcript_name(self, mock_get):
+        """
+        Verified transcripts availability when transcript name is present
```

Comment:
Will "Test that the transcripts are fetched correctly when the the transcript name is set" be more accurate?